src/cache/cache_las.c

```diff
@@ -1264,41 +1264,45 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
  *     the record and return to the caller.
  */
 int
-__wt_find_lookaside_upd(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE **updp)
+__wt_find_lookaside_upd(
+  WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE **updp, bool allow_prepare)
```

Comment:
Can you add a comment in explaining the new parameter and it's usage or when this parameter is passed as true or etc.

Reply:
fd82c8e

Added to the function documentation.